test/integration/scenarios/stress/2.register_delegate.js

```diff
@@ -17,7 +17,7 @@
 var Promise = require('bluebird');
 var lisk = require('lisk-elements').default;
 var accountFixtures = require('../../../fixtures/accounts');
-var constants = require('../../../../helpers/constants');
+const constants = require('../../../../config/mainnet/constants');
```

Comment:
Using `mainnet` constants here (and in many other places) for tests is not very consistent.

Reply:
It will be fixed by #2156